### PR TITLE
Final demonstrator

### PR DESCRIPTION
**Pull request by tbohne**

### CODE REVIEW

File: vehicle_diag_smach/interfaces/model_accessor.py [hunk 1]
```diff
@@ -5,8 +5,11 @@
 from abc import ABC, abstractmethod
 from typing import Union, Tuple
 
+import torch
```
Comment (awindler) [inline vehicle_diag_smach/interfaces/model_accessor.py:8]: Please add torch to requirements.txt

File: vehicle_diag_smach/rule_based_models/Lambdasonde.py [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+        if chan_name == "Masseleitung der Heizung der Lambdasonde":
+            return False
+        elif chan_name == "Plusleitung der Heizung der Lambdasonde":
+            return False
+        elif chan_name == "Plusleitung der Lambdasonde":
+            return True
+        elif chan_name == "Masseleitung der Lambdasonde":
+            return False
+        else:
+            # TODO: default behavior for unknown chan
```
Comment (awindler) [inline vehicle_diag_smach/rule_based_models/Lambdasonde.py:36]: Wouldn't it be better if each subcomponent had its own model class? Then you wouldn't have to solve the problem of handling unexpected or missing channel names.

File: vehicle_diag_smach/rule_based_models/Saugrohrdrucksensor.py [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+        Predicts whether the input signal comprises an anomaly.
+
+        :param input_signal: input signal to be classified
+        :param chan_name: optional name of the channel
+        :return: true -> anomaly, false -> no anomaly
+        """
+        if chan_name == "Signalleitung (Druck) des Saugrohrdrucksensors":
+            return True
+        elif chan_name == "Signalleitung (Temperatur) des Saugrohrdrucksensors":
+            return True
```
Comment (awindler) [inline vehicle_diag_smach/rule_based_models/Saugrohrdrucksensor.py:30]: Is there a reason why for some specific channels true is returned? Does it have to do with what result you would expect given the real rules and the test signals, or were the channels just selected randomly?

**Review by awindler — COMMENTED** [review]
Please update the readme file in obd_ontology to clarify how to inlcude the right version of the KG for the demonstrator.

File: vehicle_diag_smach/rule_based_models/Lambdasonde.py (hunk 2 above)
Comment (tbohne) [reply vehicle_diag_smach/rule_based_models/Lambdasonde.py:36]: I don't think so. I like the channels to be gathered under their "super component". Also, you'd still have to solve the problem then, just on a different level (when you select the subcomponent class).

File: vehicle_diag_smach/rule_based_models/Saugrohrdrucksensor.py (hunk 3 above)
Comment (tbohne) [reply vehicle_diag_smach/rule_based_models/Saugrohrdrucksensor.py:30]: At the moment this is just randomly assigned for testing purposes. As discussed, you'll implement the real version of this (cf. #32 ).

**Review by awindler — APPROVED** [review]

**Pull request merged**